treeherder/model/sql/jobs.json

```diff
@@ -349,7 +349,7 @@
 
             "sql":"SELECT s.`signature`,
                     GROUP_CONCAT(
-                        CONCAT_WS('=', s.`property`, s.`value`) SEPARATOR ',')
+                        CONCAT_WS('=', s.`property`, s.`value`) SEPARATOR '\n')
```

Comment:
I am not sure why you are switching from a ',' to a '\n'.  Could you please explain this?